### ISSUE TITLE
Changing Actor Network

### ISSUE THREAD

**Issue by junwoony** [tensorforce/tensorforce#741]
Hello,

I could find a documentation for customizing the networks for state embedding, https://tensorforce.readthedocs.io/en/latest/modules/networks.html

However, I'm wondering if there is a way to change/customize the actor network.
It looks like after creating a state embedding, `action_values` are determined by actor networks that I cannot customize or I don't have any idea how to do modify it. 

Let's say I have 3 different actions in the action space defined using MultiDiscrete of Dict spaces. My goal is to customize the actor networks for each of the possible actions, so that I can first determine two actions and use them as additional inputs to the third actor networks. So intuitively I want to make the third action depend on the other two actions.

In summary, it should look like,
```
state embedding --> 1st actor net. --> 1st action
state embedding -- >2nd actor net. --> 2nd action
state embedding + 1st and 2nd actions --> 3rd actor net. --> 3rd action
```
Any suggestions or documentation that I can take a look?
Thank you!

**Comment by AlexKuhnle** [comment]
Right now that's not easily possible, as the [provided types of policies](https://github.com/tensorforce/tensorforce/tree/master/tensorforce/core/policies) are all based on one shared network with independent parallel action heads. So currently you would need to implement your own policy object to achieve this. However, it would be possible to provide an architecture pattern like that, maybe in the form of: `network` for unconditional actions (so 1 and 2 would share the same network, different output heads), and then `conditioned_network` which takes state + unconditional actions and, as before, shares network between conditioned actions with different output heads.
